include/fiction/utils/layout_utils.hpp

```diff
@@ -408,42 +415,71 @@ CoordinateType random_coordinate(CoordinateType coordinate1, CoordinateType coor
  * @param cell_se The southeast SiQAD cell defining the ending point of the area.
  * @return A vector containing all cells within the specified area.
  */
-[[nodiscard]] inline std::vector<siqad::coord_t> all_sidbs_in_spanned_area(const siqad::coord_t& cell_nw,
-                                                                           const siqad::coord_t& cell_se) noexcept
+template <typename CoordinateType>
+[[nodiscard]] inline std::vector<CoordinateType> all_sidbs_in_spanned_area(const CoordinateType& cell_nw,
+                                                                           const CoordinateType& cell_se) noexcept
 {
-    const auto c1_cube          = siqad::to_fiction_coord<cube::coord_t>(cell_nw);
-    const auto c2_cube          = siqad::to_fiction_coord<cube::coord_t>(cell_se);
-    const auto total_cell_count = static_cast<uint64_t>(std::abs(c1_cube.x - c2_cube.x) + 1) *
-                                  static_cast<uint64_t>(std::abs(c1_cube.y - c2_cube.y) + 1);
-
-    std::vector<siqad::coord_t> all_cells{};
-    all_cells.reserve(total_cell_count);
-
-    auto current_cell = cell_nw;
-
-    // collect all cells in the area (spanned by the nw `north-west` and se `south-east` cell) going from top to down
-    // from left to right.
-    while (current_cell <= cell_se)
+    // for siqad coordinates
```

Comment:
I noticed inconsistent usage of "siqad" vs. "SiQAD" in comments. In the code, we have naming conventions of course. In the comments, we should be able to call the tool by its original spelling, no?